src/Client/Image/ImageFileUpload.php

```diff
@@ -31,7 +31,22 @@ public function __construct(string $filename)
     {
         $this->validateFile($filename);
         $this->imageFormat = self::ALLOWED_FILE_TYPES[exif_imagetype($filename)];
-        $this->imageData = iconv('UTF-8', 'UTF-8//IGNORE', utf8_encode(file_get_contents($filename)));
+        $this->imageData = $this->byteArray($filename);
+    }
+
+    /**
+     *  Generate byte array similar to the one in C# and Java
+     */
+    public function byteArray(string $filename): array
+    {
+        $data   = file_get_contents($filename);
+        $array  = array();
```

Comment:
Please make sure the code style is inline with the other files:
(Short array syntax)

run `make cs-fix` to fix automatically